cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -197,6 +197,10 @@ spec:
                   type: integer
                 activeStandby:
                   type: boolean
+                activeStanbyCount:
```

Comment:
typo: missing the `d` in `Standby`. Looks like other places in the PR need the same update.

Reply:
Oh yes. I will correct that.